yw_website/yw_website/apps/website/static/design/home_design.css

```diff
@@ -81,29 +100,59 @@
 
 
 .title {
-	text-align: center;
+	text-align: left;
 	font-size: 16px;
 	font-weight: bold;
 	color: rgb(5, 97, 92);
+	margin: 0 0;
+}
+
+.tags-label {
+	font-size: 12px;
+	margin: 10px 0 0 0;
+}
+.tags {
+	border: 1px solid rgb(5, 97, 92);;
+	display: inline-block;
+	padding: 0 5px;
```

Comment:
clickable tags is cool, should try to inline tags and put them at the top. Once again you should just go off the css on master